install.sh

```diff
@@ -3791,7 +3796,7 @@ search_pre_commit_sample_file() {
     for HIT in $(find "$START_DIR" -path "*templates/hooks/pre-commit.sample" 2>/dev/null); do
         unset IFS
 
-        HIT=$(dirname "$HIT")
+        HIT=$(dirname "$(dirname "$HIT")")
```

Comment:
Lets set what the function says, search the template folder, its not with an added `./hooks`.